NLog.Web.AspNetCore/NLog.Web.AspNetCore.csproj

```diff
@@ -5,7 +5,7 @@
 
     <TargetFrameworks>netstandard1.5;net451;net461;netstandard2.0</TargetFrameworks>
 
-    <VersionPrefix>4.5.1</VersionPrefix>
+    <VersionPrefix>4.5.2</VersionPrefix>
```

Comment:
Please update the release notes (and maybe also the changelog) when updating the version (cannot be done afterwards anymore :(.     )
Or leave the version unchanged, thanks!

Reply:
Reverted change